requirements.txt

```diff
@@ -1,15 +1,15 @@
-alembic==0.8.4
+alembic==0.8.6
 anyjson==0.3.3
 arrow==0.7.0
 dogpile.cache==0.5.7
 dogpile.core==0.4.1
-lxml==3.5.0
+lxml==3.6.0
```

Comment:
(According to the changelog, this doesn’t affect us, but it’s nice to be up to date.)